eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -405,3 +407,7 @@ case object EmptyAPIPasswordException extends RuntimeException("must set a passw
 case object IncompatibleDBException extends RuntimeException("database is not compatible with this version of eclair")
 
 case object IncompatibleNetworkDBException extends RuntimeException("network database is not compatible with this version of eclair")
+
+case class PluginTagsAndFeature(tags: Set[Int], feature: Feature) {
```

Comment:
What about `PluginParams`?
It would be more in-line with `nodeParams` and we could enrich this case class with other plugin parameters that we may want to add in the future.